back/app/services/survey_results_generator_service.rb

```diff
@@ -159,8 +159,9 @@ def select_field_query(field, as: 'answer')
   end
 
   def build_select_response(answers, field, group_field)
-    # TODO: This is an additional query for selects so performance issue here
-    question_response_count = inputs.where("custom_field_values->'#{field.key}' IS NOT NULL").count
+    nil_answer_count = answers.detect { |answer| answer[:answer].nil? }[:count]
+    question_response_count = @inputs.count - nil_answer_count
```

Comment:
Oops, I think this won't work correctly for multiselect questions since this is the number of picks and not the number of users. Going to fix this now